### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by vicpatel**
- Automation/*
- Compose/*
- System/*

Please take a look.

### CODE REVIEW

File: tests/automation/session_test.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	h.allow(types.AutomationRBACResource, "sessions.search")
+
+	var rr struct{}
+
+	h.apiInit().
+		Get(fmt.Sprintf("/sessions/%d", s.ID)).
+		Header("Accept", "application/json").
+		Expect(t).
+		Status(http.StatusOK).
+		// Assert(helpers.AssertNoErrors).
```
Comment (tjerman) [inline tests/automation/session_test.go]: Why commented?

File: tests/system/attachment_test.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	"github.com/cortezaproject/corteza-server/system/service"
+	"github.com/cortezaproject/corteza-server/system/types"
+	"github.com/cortezaproject/corteza-server/tests/helpers"
+)
+
+func (h helper) clearAttachment() {
+	h.noError(store.TruncateAttachments(context.Background(), service.DefaultStore))
+}
+
+func (h helper) repoMakeAttachment(ss ...string) *types.Attachment {
```
Comment (tjerman) [inline tests/system/attachment_test.go:22]: I do like how you've mocked attachments here :+1:

File: tests/system/role_test.go [hunk 3]
```diff
@@ -319,3 +384,40 @@ func TestRoleLabels(t *testing.T) {
...
+
+	r := h.repoMakeRole(h.randEmail())
+	memberID := id.Next()
+
+	res := h.apiInit().
+		Post(fmt.Sprintf("/roles/%d/member/%d", r.ID, memberID)).
+		Expect(t).
+		Status(http.StatusOK).
+		// Assert(helpers.AssertNoErrors).
+		// Assert(jsonpath.Len(`$.response`, 1)).
```
Comment (tjerman) [inline tests/system/role_test.go]: ?

File: tests/system/user_test.go [hunk 4]
```diff
@@ -357,6 +363,44 @@ func TestUserUpdate(t *testing.T) {
...
+	h.clearUsers()
+
+	u := h.createUserWithEmail(h.randEmail())
+	h.allow(types.UserRBACResource.AppendWildcard(), "suspend")
+
+	h.apiInit().
+		Post(fmt.Sprintf("/users/%d/unsuspend", u.ID)).
+		Expect(t).
+		Status(http.StatusOK).
+		// Assert(helpers.AssertNoErrors).
```
Comment (tjerman) [inline tests/system/user_test.go]: ?

**Review by tjerman — COMMENTED** [review]
Address those few things; then merge it.
We'll add additional bits after.

File: tests/automation/session_test.go (hunk 1 above)
Comment (vicpatel) [reply tests/automation/session_test.go]: MB, fixed it!

**Pull request merged**